style.css

```diff
@@ -234,12 +239,111 @@
     align-items: center;
   }
 
-  .grid-3-movile {
+  .projectCards {
     position: relative;
     width: 100%;
     top: 139px;
   }
 
+  .contPopUp {
+    z-index: 10;
+    display: none;
+    position: fixed;
+    padding: 10%;
+    color: black;
+    width: 100vw;
+    height: 100vh;
+    box-sizing: border-box;
+    backdrop-filter: blur(5px);
+  }
+
```

Comment:
- [ ] Great job on your popup its looking identical to the figma design however please make your popup is responsive as currently your popup content is overflowing and in the mobile version it does not scroll instead the content simply overflows from the card. One way to fix this is using responsive values (like %, rem, em etc... ) for the content sizes (width, height etc...)

Your project popup:

Desktop:

<img width="1040" alt="Screenshot 2023-05-18 at 9 56 05 AM" src="https://github.com/Zven94/portfolio/assets/44798044/ed6caef0-070c-44fd-b9b7-5884aadce0f8">

Mobile: 

<img width="361" alt="Screenshot 2023-05-18 at 10 22 37 AM" src="https://github.com/Zven94/portfolio/assets/44798044/614c2fac-1c0c-4e7d-8984-e408ef8e331a">